byway/src/main/webapp/routepage.js

```diff
@@ -29,11 +41,13 @@ function initMap() {
   }
   let map = new google.maps.Map(document.getElementById('map'), mapOptions);
   directionsRenderer.setMap(map);
-    document.getElementById("route").addEventListener("click", function() {
+  document.getElementById("route").addEventListener("click", function() {
     calcRoute(directionsService, directionsRenderer, start, end);
   });
 }
 
+
+/** Displays route overtop the map */
```

Comment:
Please add documentation on what parameter this function and others expect.

Reply:
done!